keyboards/karn/rules.mk

```diff
@@ -0,0 +1,3 @@
+SPLIT_KEYBOARD = yes
+SERIAL_DRIVER = usart
+TAP_DANCE_ENABLE = yes
```

Comment:
This can't live at the keyboard level, as it will cause compilation errors when using the QMK Configurator or any other keymap that doesn't have the necessary tap dance code.

Reply:
@waffle87 Ok thanks for letting me know. Where should I move it to? (trying to figure this out from the docs but having a hard time)

Reply:
Trying moving just the `TAP_DANCE_ENABLE` to `keyboards/karn/keymaps/colemak/rules.mk`